packages/@okta/vuepress-site/docs/guides/device-context/main/aspnet/integrationsteps.md

```diff
@@ -0,0 +1,29 @@
+### 1. Create a device ID for each device
+
+Create a device ID to identify each device. These IDs must be unique per device, or unpredictable behavior can result. For example, you can generate a UUID of the format `123e4567-e89b-12d3-a456-426655440000` in the following ways:
+
+* In a Powershell window with the command `[guid]::NewGuid()`
+* In C# with `Guid.NewGuid().ToString("N")`
+
+When you use the Embedded SDK, it's your responsibility to generate and store these device ID values.
+
+> **Note:**  The SDK expects the device ID to be exactly 32 characters long. UUIDs are 36 characters long including the hyphens.
```

Comment:
I thought we recommended a UUID but it wasn't required. Did you test a value other than a UUID?

Reply:
I didn't - no. Will ask and test

Reply:
This comment is an inform but there's no action nor is it strong enough, which considering the sentence above, isn't as useful as it could be. If the SDK requires something, make it clear (also shifting to active voice):

> **Note:**  The SDK requires exactly 32 character long device IDs. To meet this requirement, remove the hyphens ?and the ?first/last? character? of the UUID.

I don't remember is the UUID has three or four hyphens, either way, remove the hyphens first to preserve uniqueness. If there are less than four, hyphens then every character removed risks an ID clash.

Reply:
To be clear, this Note needs updating so that it the action is clear. And what is a "device ID"? The device token?